edi_project_oca/models/project_project.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _name = "project.project"
+    _inherit = ["project.project", "edi.exchange.consumer.mixin"]
+
+    edi_disable_auto = fields.Boolean()
```

Comment:
no need to re-define this flag

Reply:
I used `oca_account_edi` as example to create this module, and the field is re-declared there for `account.move`, so I did the same for consistency

Reply:
that's wrong unless you want to control the visibility by the state